woocommerce/woocommerce-android
# [TTP] Track TTP unavailable reason when TTP universal link is used

Closes: #9328


### Description

The PR adds tracking of the reason why TTP unavailable when TTP universal link is clicked (inluding the future JITM). It adds "source" property to destingvish between Payment Type Selection screen and Payment hub screen.

### Testing instructions

* Run

```
 adb shell am start -a android.intent.action.VIEW \                   
   -c android.intent.category.BROWSABLE \
   -d "https://woocommerce.com/mobile/payments/tap-to-pay"
```
on a device without NFC (e.g. emulator) or lower android version
* Notice tracking event:

```
Tracked: card_present_tap_to_pay_not_available, Properties: {"reason":"NfcNotAvailable","source":"payments_menu","plugin_slug":"woocommerce-payments","country":"unknown","blog_id":192152755,"is_wpcom_store":true,"is_debug":true}
```
